CONTRIBUTING.md

```diff
@@ -24,9 +24,54 @@ Before sending a Pull Request with significant changes, please use the [issue tr
 OpenLineage uses [GitHub's fork and pull model](https://help.github.com/articles/about-collaborative-development-models/)
 to create a contribution.
 
-Every contribution is signed to say that the contributor has the rights to make the contribution and
+Every contribution is [signed](#sign-your-work) to say that the contributor has the rights to make the contribution and
 agrees with the [Developer Certificate of Origin (DCO)](why-the-dco.md)
 
+To ensure your pull request is accepted, follow these guidelines:
+
+* All changes should be accompanied by tests
+* Do your best to have a [well-formed commit message](https://tbaggery.com/2008/04/19/a-note-about-git-commit-messages.html) for your change
+* Do your best to have a [well-formed](https://frontside.com/blog/2020-7-reasons-for-good-pull-request-descriptions) pull request description for your change
+* [Keep diffs small](https://kurtisnusbaum.medium.com/stacked-diffs-keeping-phabricator-diffs-small-d9964f4dcfa6) and self-contained
+* If your change fixes a bug, please [link the issue](https://help.github.com/articles/closing-issues-using-keywords) in your pull request description
+* Your pull request title should be of the form `[COMPONENT] Your title`, where `COMPONENT` is one of `DBT`, `AIRFLOW`, `SPARK`, etc.
```

Comment:
the issue templates separate the spec improvements proposals `[PROPOSAL] ...` from integrations `[INTEGRATION] ...` and I've been doing `[INTEGRATION] [Airflow] ...` to be more precise

Reply:
Adding `[PROPOSAL]` for an issue makes sense, but this outlines how to name you PR title. We can add `[INTEGRATION]` but I do feel it's redundant as `[AIRFLOW]` makes it clear it's for the Airflow integration. Those are just my thoughts, I'll go ahead and update the PR title format 👍